core/src/main/java/org/bitcoinj/core/PeerGroup.java

```diff
@@ -1380,13 +1476,13 @@ public void setConnectTimeoutMillis(int connectTimeoutMillis) {
      *
      * @param listener a listener for chain download events, may not be null
      */
-    public void startBlockChainDownload(PeerDataEventListener listener) {
+    public void startBlockChainDownload(ChainDownloadStartedEventListener listener) {
```

Comment:
In the context of this PR, we should stay as compatible as we can. However, we could start a new PR (which perhaps gets merged before this one) that removes PeerGroup.startBlockChainDownload() and adds a listener pair instead. I found the original ticket here: https://github.com/bitcoinj/bitcoinj/issues/464